packages/react-components/src/components/styles/ImageOverlay.style.ts

```diff
@@ -141,80 +138,60 @@ export const normalImageStyle: IStyle = {
   maxWidth: '100%'
 };
 
-/* @conditional-compile-remove(image-overlay) */
 /**
  * @private
  */
-export const brokenImageStyle = (theme: ChatTheme): IStyle => {
+export const brokenImageStyle = (theme: Theme): IStyle => {
   return {
-    color: theme.chatPalette.modalTitleWhite
+    color: theme.palette.white
   };
 };
 
-/* @conditional-compile-remove(image-overlay) */
 /**
  * @private
  */
-export const closeButtonStyles = (theme: ChatTheme): IStyle => {
+export const closeButtonStyles = (theme: Theme): IStyle => {
   return {
-    color: theme.chatPalette.modalTitleWhite,
+    color: theme.palette.white,
     ':hover': {
-      color: theme.chatPalette.modalTitleWhite,
-      backgroundColor: theme.chatPalette.modalButtonBackgroundHover
```

Comment:
we don't want to update background colours anymore?

Reply:
> we don't want to update background colours anymore?

Hi John, now that ImageOverlay is wrapped in it's own theme provider with the new ImageOverlayTheme, Contosos can change the background color along with other colors through override the ImageOverlayTheme theme.